src/compiler/reverse.jl

```diff
@@ -279,7 +274,7 @@ xaccum(ir, xs...) = push!(ir, xcall(Zygote, :accum, xs...))
 
 function passthrough_expr(ex::Expr)
     # Metadata we want to preserve
-    isexpr(ex, GlobalRef, :call, :isdefined, :inbounds, :meta, :loopinfo) && return true
+    isexpr(ex, GlobalRef, :call, :isdefined, :inbounds, :meta, :loopinfo, :enter, :leave, :catch) && return true
```

Comment:
This should work on older version of julia that do not have `enter` and `leave`.
But the feature will not

Reply:
Zygote has compat for Julia 1.6 which already has `enter` and `leave` to represent exception handlers so it should be fine.

Reply:
oh nice, i thought they were way newer